### PR TITLE
16.0 mig-account_payment_notification

### PR DESCRIPTION
**Pull request by anddago78**
This module lets you notify partners about sent payment or debit orders.

You can do that by e-mail and/or by SMS. It is automated by default.

MT-3084 @moduon @Shide @rafaelbn @fcvalgar please reviews :)

### CODE REVIEW

**Review by fcvalgar — APPROVED** [review]
Functional review.
LGTM, thank you @anddago78 

image

**Review by rafaelbn — APPROVED** [review]
👍🏼 Maybe we could improve a little bit templates

@fcvalgar will check again and after we decide

This migration is 👍🏼 exactly like v15

File: account_payment_notification/README.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+.. |maintainer-yajo| image:: https://github.com/yajo.png?size=40px
+    :target: https://github.com/yajo
+    :alt: yajo
```
Comment (rafaelbn) [inline account_payment_notification/README.rst:121]: @anddago78 add mysel here please

:target: github.com/rafaelbn
:alt: rafaelbn

Thanks

**Review by rafaelbn — APPROVED** [review]

**Comment by rafaelbn** [comment]
@yajo ready to merge!

**Comment by rafaelbn** [comment]
Please @OCA/banking-maintainers could you please merge? This is a v16 migration we are using un PRODUCTION already for serveral months in 16 and sin 2022 en 15

Thank you! 😄

**Pull request merged**